org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterGenericButton.java

```diff
@@ -117,7 +126,7 @@ public void commandReceived(ZclCommand command) {
     }
 
     @Override
-    public void attributeUpdated(ZclAttribute attribute) {
+    public void attributeUpdated(ZclAttribute attribute, Object val) {
```

Comment:
`val` has to be used in `AttributeReportSpec.matches()`.
I think changing `boolean matches(ZclAttribute attribute)` to `boolean matches(ZclAttribute attribute, Object value)` should be fine.
(I can propose a PR next week)